### DISCUSSION TITLE
3.3.0 release

### DISCUSSION THREAD

**Discussion by matthewhegarty** [django-import-export/django-import-export#1620] [General]
hi All

I'm bringing together some PRs for a forthcoming release (3.3.0).

Any reviews of [existing PRs](https://github.com/django-import-export/django-import-export/pulls) would be very welcome.

**Comment by matthewhegarty** [comment]
3.3.0 released.  Thanks to all who contributed and reviewed.

**Reply by mgrdcm** [reply]
And thanks to you for getting the release out!
